src/main/resources/default/templates/biz/jobs/params/lookuptable.html.pasta

```diff
@@ -9,4 +9,26 @@
                label="@param.getLabel()"
                help="@param.getDescription()"
                optional="true"
-               value="@param.getBuilder().as(sirius.biz.codelists.LookupTableParameter.class).createLookupValue(param.get(context).orElse(null).as(String.class))"/>
+               value="@param.getBuilder().as(sirius.biz.codelists.LookupTableParameter.class).createLookupValue(param.get(context).orElse(null).as(String.class))">
+    <i:block name="script">
+        const _elem = sirius.querySelector(".param-___param.getName()");
+        _elem.addEventListener("clear", function () {
+            autocomplete.select.clear(true, false);
+        });
+        _elem.addEventListener("updated-value", function (event) {
+            autocomplete.val(event.detail, true);
+        });
+    </i:block>
+
+    <i:if test="isFilled(suggestionUri)">
+        <i:block name="autocompleteOptions">
+            {
+                suggestionsUriBuilder: function (query) {
+                    const context = sirius.getFormData(sirius.querySelector(".edit-form"));
+                    return "@suggestionUri" + "?query="
+                            + query + "&" + new URLSearchParams(context).toString();
```

Comment:
Why not add the query to the search params and append the toString directly after the questionmark? seems a bit less "hacky"